src/annotator/frame-observer.js

```diff
@@ -171,7 +173,7 @@ export function onDocumentReady(frame, callback, { pollInterval = 10 } = {}) {
 
   const cancelPoll = () => {
     clearTimeout(pollTimer);
-    pollTimer = null;
+    pollTimer = undefined;
```

Comment:
`clearTimeout` expects `number|undefined`, so it was more convenient to use `undefined` as the null value for `pollTimer`.